From: Jordi Mallach <jordi@debian.org>
Description: test for alsactl
 Test for @sbindir@/alsactl to fix state restoration for systems with
 split /usr.
Forwarded: no

Index: alsa-utils-1.0.25/alsactl/90-alsa-restore.rules.in
===================================================================
--- alsa-utils-1.0.25.orig/alsactl/90-alsa-restore.rules.in	2012-01-25 10:43:38.000000000 +0100
+++ alsa-utils-1.0.25/alsactl/90-alsa-restore.rules.in	2013-01-10 01:12:15.679998854 +0100
@@ -1,2 +1,2 @@
 ACTION=="add", SUBSYSTEM=="sound", KERNEL=="controlC*", KERNELS=="card*", \
-        RUN+="@sbindir@/alsactl restore $attr{number}"
+        TEST=="@sbindir@/alsactl", RUN+="@sbindir@/alsactl restore $attr{number}"
